proto/v1/compliance.proto

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grafeas.v1;
+
+import "google/protobuf/timestamp.proto";
+import "proto/v1/cvss.proto";
+import "proto/v1/package.proto";
+import "proto/v1/vulnerability.proto";
+
+option go_package = "github.com/grafeas/grafeas/proto/v1/grafeas_go_proto";
+option java_multiple_files = true;
+option java_package = "io.grafeas.v1";
+option objc_class_prefix = "GRA";
+
+message ComplianceNote {
+  // The OS and config versions the benchmark applies to.
```

Comment:
is 1 intentionally skipped?

Reply:
There should be an "id" field before this with the number 1 as per the design.

Reply:
I did not include the id since our general pattern is to store the occurrenceId / note Id in the grafeas proto. https://github.com/grafeas/grafeas/blob/master/proto/v1/grafeas.proto#L194

Lmk if you think we still should include it here. Similar reason for the Note as well.

Reply:
We don't store the whole Note/Occurrence in the CIS scanner, just the ComplianceNote/Occurrence part of it, and for the instance scanning integration, we, we also just look at the ComplianceNotes/Occurrences. So there we'd need the ID in this message in order to connect the non-compliance report with the benchmark that was found non-compliant.

Reply:
We decided to create a new message / struct to return the compliance occurrence and id.